src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestHelpers/TestCase.cs

```diff
@@ -8,7 +8,10 @@
 
 namespace Microsoft.PowerFx.Core.Tests
 {
-    internal class TestCase
+    /// <summary>
+    ///  Describe a test case from the .txt file.
+    /// </summary>
+    public class TestCase
     {
         // Formula string to run 
         public string Input;
```

Comment:
Should we be maintaining this Dict indexed by runner name? It seems like the goal of this PR is to have just one expected result loaded at a time, based on what's been overriden.

Reply:
exactly. I mentioned that in the other comments. I wanted to get validation of the direction before ripping it out wholesale.

Reply:
We first need to migrate other backends away from /*engine*/ and then we can remove this.